app/src/main/java/com/prangesoftwaresolutions/audioanchor/PlayActivity.java

```diff
@@ -128,6 +128,7 @@ protected void onCreate(Bundle savedInstanceState) {
         mLastSleepTime = sharedPreferences.getInt(getString(R.string.preference_last_sleep_key), Integer.valueOf(getString(R.string.preference_last_sleep_val)));
         mDirectory = sharedPreferences.getString(getString(R.string.preference_filename), null);
 
+
```

Comment:
Same as mentioned in the comment for the MediaPlayerService. Don't include this file in your commit.